test/fake.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Suggest move this file to root directory instead of `test`, rename to `test_chrome_key.pem`